### ISSUE TITLE
feat(headerComponent): use customElement within/as headerComponent

### ISSUE THREAD

**Issue by mroseboom** [ceolter/ag-grid-aurelia#15]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid-aurelia/blob/master/CONTRIBUTING.md#question
```
**What is the motivation / use case for changing the behavior?**
Is it (maybe already) possible to use a customElement within/as a HeaderComponent, just like a cellEditor?

I'm using still the old way through `header-cell-template`, but that is dropped in v9.

See underlying message:

> ag-Grid: since v8, custom headers are now done using components. Please refer to the documentation https://www.ag-grid.com/javascript-grid-header-rendering/. Support for the old way will be dropped in v9.

* **ag-grid-aurelia version:** 8.2.0

* **ag-Grid/ag-Grid-Enterprise version:** 8.2.0

**Comment by dmarini** [comment]
Any word on this? I noticed that the Angular wrapper has something that appears to support declarative header cell templates, was wondering if that's on the books for the aurelia wrapper

**Comment by mroseboom** [comment]
I will try to make a PR for this. @dmarini could you point me to the angular source code or give a link?

**Comment by swalters** [comment]
@mroseboom wait until @seanlandsman gives you a thumbs up.  He may be in the middle of coding this

**Comment by seanlandsman** [comment]
Hi,

Thanks for raising this. I've raised AG-776 our side to track this - you can view its progress on our pipeline page here: https://www.ag-grid.com/ag-grid-pipeline/

We'll update this thread with updates as soon as we have any.

thanks

**Comment by seanlandsman** [comment]
@mroseboom @swalters I'll be starting work on this (and the other issues) today - apologies for the delay

**Comment by seanlandsman** [comment]
This has now been implemented - it'll be available in our next release

thanks

**Comment by JohnMasterson** [comment]
Just to confirm that this was included in version 14.0.0 which is now available.
